src/components/interfaces/MOBILE_API.xml

```diff
@@ -3201,11 +3208,20 @@
                 True: Available, False: Not Available, Not present: Not Available.
             </description>
         </param>
-        <param name="availableHDsAvailable" type="Boolean" mandatory="false">
+        <param name="availableHDsAvailable" type="Boolean" mandatory="false" deprecated="true" since="5.0">
             <description>
                 Availability of the getting the number of available HD channels.
                 True: Available, False: Not Available, Not present: Not Available.
             </description>
+            <history>
+                <param name="availableHDsAvailable" type="Boolean" mandatory="false" since="1.0" until="5.1"/>
+            </history>
+        </param>
+        <param name="availableHdChannelsAvailable" type="Boolean" mandatory="false" since="5.0">
```

Comment:
since="5.2"?

Reply:
@yang1070 Fixed in 4d29747d10ac9d7d7d4294f7a7be4c35ee132c75